scheds/include/scx/common.bpf.h

```diff
@@ -389,13 +388,14 @@ static inline u32 log2_u32(u32 v)
  */
 static inline u32 log2_u64(u64 v)
 {
-        u32 hi = v >> 32;
-        if (hi)
-                return log2_u32(hi) + 32 + 1;
-        else
-                return log2_u32(v) + 1;
+	u32 hi = v >> 32;
+	if (hi)
+		return log2_u32(hi) + 32 + 1;
+	else
+		return log2_u32(v) + 1;
 }
 
 #include "compat.bpf.h"
 
 #endif	/* __SCX_COMMON_BPF_H */
+
```

Comment:
Can you include only the changes to `scheds/rust/scx_bpfland`?

Reply:
Yes. I rolled back most of the cosmetic stuff within and outside of main.bpf.c.
Thank you for the suggestion.